projects/demo/src/modules/experimental/button/button.template.html

```diff
@@ -41,6 +41,11 @@
             heading="Loading"
             [content]="example4"
         >
+            <tui-notification class="tui-space_bottom-4">
+                Pass a string as
+                <code>loading</code>
+                for a18n indication
```

Comment:
Did you mean `a11y` ?

```suggestion
                for a11y indication
```